clang_build/target.py

```diff
@@ -312,24 +312,24 @@ def link(self):
 class Executable(Compilable):
     def __init__(self,
             name,
-            working_directory,
-            buildDirectory,
+            root_dir,
```

Comment:
It is `root_dir`but `include_directories`. That seems inconsistent.

Reply:
Directory variable names have now been changed with 72fc91a.